src/js/view/controls/components/menu/settings-menu.js

```diff
@@ -314,6 +314,12 @@ class SettingsMenu extends Menu {
             gearButton.setAttribute('aria-expanded', true);
         }
 
+        if (document.querySelector('.jw-breakpoint-0') || document.querySelector('.jw-breakpoint-1')) {
+            const mediaState = this.model._mediaModel.attributes.mediaState;
+            this.mediaStateWhenOpened = mediaState;
```

Comment:
I think it might be good to add an `_` to the start of this property as we only want to use this internally.

Reply:
instead use model.get('state')